ui/src/app/app.tsx

```diff
@@ -85,9 +85,9 @@ requests.onError.subscribe(async err => {
             .getAttribute('href')
             .replace(/\/$/, '');
         if (isSSO) {
-            window.location.href = `${basehref}/auth/login?return_url=${encodeURIComponent(location.href)}`;
```

Comment:
It appears that this change is not required for SSO flow. Looks like `history.push` taking into account page basehref but `window.location.href` don't. so we still need to has set `basehref` manually here. Can you please revert this line?